test/bootc-sources/microshift_repo_config.sh.template

```diff
@@ -5,6 +5,7 @@ USHIFT_LOCAL_REPO_FILE=/etc/yum.repos.d/microshift-local.repo
 OCP_MIRROR_REPO_FILE=/etc/yum.repos.d/openshift-mirror-beta.repo
 OCP_DTPATH_REPO_FILE=/etc/yum.repos.d/openshift-fast-datapath.repo
 OCP_RHOCP_REPO_FILE=/etc/yum.repos.d/openshift-rhocp.repo
+CENTOS_NFV_SIG_MIRROR_REPO_FILE=/etc/yum.repos.d/centos-sig-nfv.repo
```

Comment:
We're deleting microshift* and openshift* repos in the cleanup. Please, name with repo with `microshift-` prefix.

Reply:
Ack. Done.